platform/android/java/lib/src/org/godotengine/godot/io/StorageScope.kt

```diff
@@ -90,6 +90,15 @@ internal enum class StorageScope {
 				return APP
 			}
 
+			var rootDir: String? = System.getenv("ANDROID_ROOT")
+			if (rootDir != null && canonicalPathFile.startsWith(rootDir)) {
+				return APP
+			}
+
+			if (canonicalPathFile.startsWith("/system/")) {
+				return APP
+			}
```

Comment:
Aren't `/system` and `System.getenv("ANDROID_ROOT")` the same value? Do we need this second check?

Reply:
Seems like it is the same, removed extra `/system` check.